### PR TITLE
fix: use h2 for heading and h3 for price

### PR DESCRIPTION
**Pull request by jamesgorrie**
The [axe plugin we're using](https://github.com/dequelabs/axe-core-npm) was reporting that [we had the headers in the wrong order](https://dequeuniversity.com/rules/axe/4.4/heading-order?application=axeAPI).

After re-assessing the markup, I have opted to use sections for each product and an H2 for it's name, and remove the heading for the price, which didn't seem to make semantic sense.

**Before**

https://github.com/guardian/support-frontend/assets/31692/df9eeed0-6985-49c9-b2f2-81beeb3dff1d

**After**

https://github.com/guardian/support-frontend/assets/31692/848e7377-6b6b-4d23-937c-26e813e18518

### CODE REVIEW

File: support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx [hunk 1]
```diff
@@ -201,13 +201,15 @@ export function ThreeTierCard({
 	const promoPriceCopy = `${currency}${promoPrice}/${recurringContributionPeriodMap[paymentFrequency]}`;
 	const quantumMetricButtonRef = `tier-${cardTier}-button`;
 	return (
-		<div css={container(isRecommended, isUserSelected, isRecommendedSubdued)}>
+		<section
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx:204]: Presume as we grouped together everything inside to a TierCard.

File: support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx [hunk 2]
```diff
@@ -201,13 +201,15 @@ export function ThreeTierCard({
...
+			css={container(isRecommended, isUserSelected, isRecommendedSubdued)}
+		>
 			{isUserSelected && <ThreeTierLozenge title="Your selection" />}
 			{isRecommended && !isUserSelected && (
 				<ThreeTierLozenge subdue={isRecommendedSubdued} title="Recommended" />
 			)}
-			<h3 css={titleCss}>{title}</h3>
-			<h2 css={priceCss(!!planCost.discount)}>
+			<h2 css={titleCss}>{title}</h2>
+			<p css={priceCss(!!planCost.discount)}>
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx:212]: Yep not a header really so move up the h2

**Review by paul-daniel-dempsey — APPROVED** [review]
An definate accessibility improvement

File: support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx (hunk 1 above)
Comment (jamesgorrie) [reply support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx:204]: I was thinking these might actually be better as a list with a heading "Products".

I have used a process previously which is to strip all CSS and see if your page makes sense. Fancy pairing on that?

**Pull request merged**